alerter.js

```diff
@@ -24,3 +24,22 @@ alertInCelcius(400.5);
 alertInCelcius(303.6);
 console.log(`${alertFailureCount} alerts failed.`);
 console.log('All is well (maybe!)');
+
+//Testing Alerter
+let threshold = 100;
+let actualAlertFailureCount = 0;
+
+function testAlerter(farenheit) {

```

Comment:
stub/test code and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.